sdk/python/lib/pulumi/dynamic/dynamic.py

```diff
@@ -240,6 +253,12 @@ def delete(self, _id: str, _props: Dict[str, Any]) -> None:
         assumed to still exist.
         """
 
+    def configure(self, req: ConfigureRequest) -> None:
```

Comment:
Happy either way on positional args, though I personally prefer the request object pattern -- @justinvp, thoughts?

I *do* think we should return an empty `ConfigureResult` just to give ourselves optionality in future.

Reply:
I started adding it `ConfigureResult` and it feels a bit weird since it's empty. If we ever need to return something in the future, it can be optional (since today there is no data to put in there). That means wan have the method return `None` now, and if we require it in the future we can return `Optional[ConfigureResult]`.

Reply:
Also delete currently returns `None`